src/main/java/wooteco/subway/service/member/FavoriteService.java

```diff
@@ -0,0 +1,60 @@
+package wooteco.subway.service.member;
+
+import org.springframework.stereotype.Service;
+import wooteco.subway.domain.member.Favorite;
+import wooteco.subway.domain.member.Favorites;
+import wooteco.subway.domain.member.Member;
+import wooteco.subway.domain.member.MemberRepository;
+import wooteco.subway.service.exception.SameStationException;
+import wooteco.subway.service.member.dto.FavoriteRequest;
+import wooteco.subway.service.member.dto.FavoriteResponse;
+import wooteco.subway.service.station.StationService;
+
+import java.util.List;
+
+@Service
+public class FavoriteService {
+    private MemberRepository memberRepository;
+    private StationService stationService;
+
+    public FavoriteService(MemberRepository memberRepository, StationService stationService) {
+        this.memberRepository = memberRepository;
+        this.stationService = stationService;
+    }
+
+    public void createFavorite(FavoriteRequest favoriteRequest, Member member) {
+        validate(favoriteRequest);
+        Long sourceId = stationService.findStationByName(favoriteRequest.getSourceName()).getId();
+        Long destinationId = stationService.findStationByName(favoriteRequest.getDestinationName()).getId();
+        member.addFavorite(new Favorite(sourceId, destinationId));
+        memberRepository.save(member);
+    }
+
+    public void removeFavorite(FavoriteRequest favoriteRequest, Member member) {
+        validate(favoriteRequest);
+        Long sourceId = stationService.findStationByName(favoriteRequest.getSourceName()).getId();
+        Long destinationId = stationService.findStationByName(favoriteRequest.getDestinationName()).getId();
+        member.removeFavoriteById(sourceId, destinationId);
```

Comment:
```suggestion
        member.removeFavorite(new Favorite(sourceId, destinationId));
```
member.addFavorite 메서드처럼 Favorite 객체로 메시지 주고 받는게 어떨까요?